packages/tldraw/src/lib/ui/hooks/useCommandBarActions.ts

```diff
@@ -0,0 +1,32 @@
+import { unwrapLabel, useActions } from '../context/actions'
+import { TLUiTranslationKey } from './useTranslation/TLUiTranslationKey'
+import { useTranslation } from './useTranslation/useTranslation'
+
+/** @public */
+export function useCommandBarActions(search: string) {
+	const actions = useActions()
+	const msg = useTranslation()
+	return Object.values(actions)
+		.filter((action) => {
+			// Don't show actions until we start searching
+			if (search === '') return false
```

Comment:
i think part of the benefit of a cmd-k palette is to show popular actions at the top to spur ideas. it's not meant to be really like a cmd-space (spotlight on mac) type of feeling. i think typically cmd+k should show some default actions but happy to debate this!

Reply:
Yeah. One idea I had was that we could show actions that users never used before. That said we'd need to store which ones they did use to implement that. Which is kinda problematic since we don't have auth 😄 

Another option would be to show the ones that the users are most likely to use. If you have a group selected there's a high chance you might want to ungroup it,...

Or just a predetermined set of actions we want to expose. Could be a good way of making people aware of new features if we always added new actions to the list.

Reply:
we can make it smarter over time but having just a predetermined set to start with might be good. even just using our vercel analytics on highest used actions would be fine place to start.